program_1.rb

```diff
@@ -0,0 +1,71 @@
+#1. Write a Ruby program to check whether a given value appears everywhere in a given array. A value is "everywhere" in an array if it presents for #every pair of adjacent elements in the array.
+
+array_1 = []
+while true
+	puts "enter array elements"
+	ele = gets.chop.to_s
+	if ele ==""
+		break
+	end
+	array_1.push(ele)
```

Comment:
Fix indentation.

Reply:
How should i indent them? I usually use the tab key once.